common/model/opening-hours.js

```diff
@@ -1,77 +1,103 @@
 // @flow
 function createPlace(data: Place) { return (data: Place); }
 
-type OpeningHoursDay = {|
-  dayOfWeek: string,
+type Days = 'Monday' | 'Tuesday' | 'Wednesday' | 'Thursday' | 'Friday' | 'Saturday' | 'Sunday';
+
+export type OpeningHoursDay = {
+  dayOfWeek: Days,
   closes: string,
   opens?: string,
   note?: string
-|};
+};
+
+type ExceptionalOpeningHoursDay = OpeningHoursDay & {
+  overrideDate: Date
+}
 
-export type OpeningHours = Array<OpeningHoursDay>;
+export type OpeningHours = {
+  openingHours: OpeningHoursDay[],
+  exceptionalOpeningHours?: ExceptionalOpeningHoursDay[]
+}
 
 type Place = {|
   id: string,
   name: string,
-  openingHours: OpeningHours
+  openingHours: OpeningHoursDay[],
 |}
 
 export type PlacesOpeningHours = Array<Place>;
 
-export const galleryOpeningHours: OpeningHours = [
-  {dayOfWeek: 'Monday',    closes: '00:00', note: 'Galleries closed'},
-  {dayOfWeek: 'Tuesday',   opens: '10:00', closes: '18:00'},
-  {dayOfWeek: 'Wednesday', opens: '10:00', closes: '18:00'},
-  {dayOfWeek: 'Thursday',  opens: '10:00', closes: '22:00'},
-  {dayOfWeek: 'Friday',    opens: '10:00', closes: '18:00'},
-  {dayOfWeek: 'Saturday',  opens: '10:00', closes: '18:00'},
-  {dayOfWeek: 'Sunday',    opens: '11:00', closes: '18:00'}
-];
+export const galleryOpeningHours: OpeningHours = {
+  openingHours: [
```

Comment:
Would these be more readable as e.g.
```
const galleryOpeningHours: {
  standard: [],
  exceptional: []
}
```

(i.e. without the slightly weird repetition of 'openingHours')?

Reply:
Yep, I've started doing this with the work on the opening times alert. I'll leave it in there